spring-boot/current-community/cache/booster.yaml

```diff
@@ -2,10 +2,10 @@ metadata:
   app:
     launcher:
       runsOn:
-        - '!starter'
+      - '!starter'
```

Comment:
This seems weird, are you sure it's below `runsOn`?

Reply:
I didn't touch that, the YAML parser did so I assume it did what it thought was correct but let me check nevertheless…

Reply:
According to https://www.json2yaml.com/, it seems OK. (I hate YAML for this kind of crap)

Reply:
It seems that it is generated below `launcher` but if you say it's ok, I'll take your word on it :)